### PR TITLE
Add note to README directing viewers to the new k8ssandra based repo

### PR DESCRIPTION
**Pull request by jdonenine**

### CODE REVIEW

**Review by johnsmartco — CHANGES_REQUESTED** [review]
Minor: in text, init cap K8ssandra.  Keep it lowercase k8ssandra in GH repo paths, of course. Otherwise, looks good. Thanks!

**Comment by jdonenine** [comment]
> Minor: in text, init cap K8ssandra. Keep it lowercase k8ssandra in GH repo paths, of course. Otherwise, looks good. Thanks!

Thanks, I should have caught that.  Done!

**Review by johnsmartco — APPROVED** [review]
lgtm

**Review by bradfordcp — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 The DataStax Kubernetes Operator for Apache Cassandra&reg;
 
+:warning: ***Ongoing development of cass-operator and future releases have been migrated to the K8ssandra project.  Head over to the new [repo](https://github.com/k8ssandra/cass-operator) for more information on using cass-operator!***. :warning:
```
Comment (bradfordcp) [inline README.md:8]:
```suggestion
:warning: ***Ongoing development of cass-operator and future releases have been migrated to the K8ssandra project.  Head over to the new repo [k8ssandra/cass-operator](https://github.com/k8ssandra/cass-operator) for more information on using cass-operator!***. :warning:
```

**Review by bradfordcp — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bradfordcp) [reply README.md:8]: This can be a follow-up

**Pull request merged**